Jenkinsfiles/release_pieline/Jenkinsfile

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+pipeline { 
+  agent none
+  parameters {
+    string(
+      name: 'BRANCH',
+      defaultValue:"master",
+      description: "Kata Containers Branch"
+    )
+    string(
+      name: 'NEW_VERSION',
+      defaultValue:"",
+      description: "Kata Containers version"
+    )
+  }
+  environment {
+    PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/local/go/bin"
```

Comment:
Nit: We probably don't need `/usr/local/games` :smile: :video_game: :space_invader: